minikube-sample/main.ts

```diff
@@ -8,12 +8,35 @@ const workspaceStore = Store.workspaceStore
 
 export default class MinikubeExtension extends LensMainExtension {
 
+  syncTimer: NodeJS.Timeout
+
   async onActivate(): Promise<void> {
     console.log("minikube extension activated")
 
     this.syncMinikube()
   }
 
+  async onDeactivate(): Promise<void> {
+    console.log("minikube extension deactivated")
+
+    if (this.syncTimer) {
+      clearTimeout(this.syncTimer);
+    }
```

Comment:
`clearTimeout` already handles this case.
```suggestion
    clearTimeout(this.syncTimer);
```